src/state/trajectory/actions.ts

```diff
@@ -4,32 +4,27 @@ import {
     RECEIVE_AGENT_IDS,
     RECEIVE_AGENT_NAMES,
     RECEIVE_SIMULARIUM_FILE,
-    SET_SIMULARIUM_CONTROLLER,
     LOAD_LOCAL_FILE_IN_VIEWER,
-    SET_VIEWER_STATUS,
     LOAD_NETWORKED_FILE_IN_VIEWER,
     REQUEST_PLOT_DATA,
     CLEAR_SIMULARIUM_FILE,
     LOAD_FILE_VIA_URL,
 } from "./constants";
 import {
-    MetadataStateBranch,
+    TrajectoryStateBranch,
     ReceiveAction,
     RequestAction,
-    SetSimulariumControllerAction,
     LocalSimFile,
     NetworkedSimFile,
     RequestCachedPlotAction,
-    ViewerStatusInfo,
-    SetViewerStatusAction,
     RequestNetworkFileAction,
     RequestLocalFileAction,
     ClearSimFileDataAction,
     LoadViaUrlAction,
 } from "./types";
 import { SimulariumController } from "@aics/simularium-viewer/type-declarations";
 
-export function receiveMetadata(payload: MetadataStateBranch): ReceiveAction {
+export function receiveMetadata(payload: TrajectoryStateBranch): ReceiveAction {
```

Comment:
is this being used? should probably be named something more specific

Reply:
Yeah, `receiveMetadata` is used. Would `receiveTrajectoryMetadata` be better? I might do that in a separate PR though.

Reply:
I think it should be renamed in this work, you're renaming the branch from 'metadata' to 'trajectory' so anything that references metadata should now reference trajecory. I think it could just be called "recieveTrajectory"

Reply:
Okay. The reason I thought something like `receiveTrajectoryMetadata` would be better than `receiveTrajectory` is because we have a `receiveSimulariumFile` action in the trajectory state branch as well. I thought having both `receiveTrajectory` and `receiveSimulariumFile` might sound like it's the same thing when `receiveTrajectory` receives the metadata only (stuff from `TrajectoryFileInfo`) while `receiveSimulariumFile` receives a `simulariumFile` object which has `name`, `data`, and `lastModified`. Do you still think `receiveTrajectory` is better to use since it matches the name of the state branch?

Reply:
I just renamed a bunch of variables containing `metadata` (including `receiveMetadata`).